### ISSUE TITLE
where you find the Coco label？

### ISSUE THREAD

**Issue by wwwloewww** [gimpong/CVPR25-Condenser#2]
Could you please tell me where you got Coco_Trainlabel and Coco_Vallabel?
From annotations_trainval2014, I can just find captions of the images

**Comment by gimpong** [comment]
Hello! 🤗

You may download them from [Coco_Trainlabel](https://drive.google.com/file/d/1cwup51kcr4m7v9jO14ArpxKMA4O3-Uge/view?usp=sharing) and [Coco_Vallabel](https://drive.google.com/file/d/1PNw4U3T2MhzAEBWGGgceXvYU3cZ7mJL1/view?usp=sharing), respectively. 

Source: [Volumetric Aggregation Transformer](https://github.com/Seokju-Cho/Volumetric-Aggregation-Transformer).

Best.

**Comment by wwwloewww** [comment]
> Hello! 🤗
> 
> You may download them from [Coco_Trainlabel](https://drive.google.com/file/d/1cwup51kcr4m7v9jO14ArpxKMA4O3-Uge/view?usp=sharing) and [Coco_Vallabel](https://drive.google.com/file/d/1PNw4U3T2MhzAEBWGGgceXvYU3cZ7mJL1/view?usp=sharing), respectively.
> 
> Source: [Volumetric Aggregation Transformer](https://github.com/Seokju-Cho/Volumetric-Aggregation-Transformer).
> 
> Best.

Ohhhh,  thank you so much! And is there the google drive link of ImageNet label dataset？I am extremely grateful！

**Comment by gimpong** [comment]
You may see here: [Google Drive](https://drive.google.com/file/d/10zxG2VExoEZUeyQl_uXga2OWHjGeZaf2/view?usp=sharing). Also from [Volumetric Aggregation Transformer](https://github.com/Seokju-Cho/Volumetric-Aggregation-Transformer). :)

**Comment by wwwloewww** [comment]
> You may see here: [Google Drive](https://drive.google.com/file/d/10zxG2VExoEZUeyQl_uXga2OWHjGeZaf2/view?usp=sharing). Also from [Volumetric Aggregation Transformer](https://github.com/Seokju-Cho/Volumetric-Aggregation-Transformer). :)

oh, thank you so much, but i want the label data of ImageNet as the picture, not the PASCAL, hahahahaha.

![Image](https://github.com/user-attachments/assets/3ee4ba61-9021-4606-a290-32d6b388c45b)

**Comment by gimpong** [comment]
I see. 😂 ImageNet is used to evaluate the colorization task, where `data` is grayscale images, while `label` is colored counterparts. You can convert color images into grayscale images manually.

**Comment by wwwloewww** [comment]
> I see. 😂 ImageNet is used to evaluate the colorization task, where `data` is grayscale images, while `label` is colored counterparts. You can convert color images into grayscale images manually.

oookok, i see. Thank you so much!

**Comment by wwwloewww** [comment]
Excuse me. Could you please provide the google drive links for the data of the ImageNet test set and training set? I feel that the years I downloaded might not match

**Comment by gimpong** [comment]
Hello and thanks for the feedback! We are preparing for it. We will let you know when ready (together with #3 , where there seems to be some compatibility issues), maybe in 2-3 days. You can skip them first and try other experiments.

> cc @luotc-why

**Comment by wwwloewww** [comment]
> Hello and thanks for the feedback! We are preparing for it. We will let you know when ready (together with [#3](https://github.com/gimpong/CVPR25-Condenser/issues/3) , where there seems to be some compatibility issues), maybe in 2-3 days. You can skip them first and try other experiments.
> 
> > cc [@luotc-why](https://github.com/luotc-why)

okok, thank you so much!

**Comment by wwwloewww** [comment]
> Hello and thanks for the feedback! We are preparing for it. We will let you know when ready (together with [#3](https://github.com/gimpong/CVPR25-Condenser/issues/3) , where there seems to be some compatibility issues), maybe in 2-3 days. You can skip them first and try other experiments.
> 
> > cc [@luotc-why](https://github.com/luotc-why)

Hello, thank you for your high-quality work. Is your imagenet dataset ready? If it is, I would be extremely grateful!

**Comment by luotc-why** [comment]
> > Hello and thanks for the feedback! We are preparing for it. We will let you know when ready (together with [#3](https://github.com/gimpong/CVPR25-Condenser/issues/3) , where there seems to be some compatibility issues), maybe in 2-3 days. You can skip them first and try other experiments.
> > > cc [@luotc-why](https://github.com/luotc-why)
> 
> Hello, thank you for your high-quality work. Is your imagenet dataset ready? If it is, I would be extremely grateful!

Hello, we’ve uploaded the 50k ImageNet train/test split labels for the coloring task. You can find them at the following link:
https://huggingface.co/datasets/why-HH/vicl_imagenet_labels/tree/main.

Please feel free to reach out if you have any questions or need further assistance!

**Comment by wwwloewww** [comment]
> > > Hello and thanks for the feedback! We are preparing for it. We will let you know when ready (together with [#3](https://github.com/gimpong/CVPR25-Condenser/issues/3) , where there seems to be some compatibility issues), maybe in 2-3 days. You can skip them first and try other experiments.
> > > > cc [@luotc-why](https://github.com/luotc-why)
> > 
> > 
> > Hello, thank you for your high-quality work. Is your imagenet dataset ready? If it is, I would be extremely grateful!
> 
> Hello, we’ve uploaded the 50k ImageNet train/test split labels for the coloring task. You can find them at the following link: https://huggingface.co/datasets/why-HH/vicl_imagenet_labels/tree/main.
> 
> Please feel free to reach out if you have any questions or need further assistance!

Thank you so much! thank you for your high-quality work!
